doc/classes/TabContainer.xml

```diff
@@ -4,8 +4,9 @@
 		Tabbed container.
 	</brief_description>
 	<description>
-		Sets the active tab's [code]visible[/code] property to the value [code]true[/code]. Sets all other children's to [code]false[/code].
+		Arranges [Control] children into a tabbed view, creating a tab for each one. The active tab's corresponding [Control] has its [code]visible[/code] property set to [code]true[/code], and all other childrens' to [code]false[/code].
```

Comment:
```suggestion
		Arranges [Control] children into a tabbed view, creating a tab for each one. The active tab's corresponding [Control] has its [code]visible[/code] property set to [code]true[/code], and all other children's to [code]false[/code].
```

It was correct before. The plural of "child" is "children", the `'s` is the possessive marker.